test/tests/functions/parsed/tests

```diff
@@ -63,4 +63,13 @@
     requirement = "The ParsedFunction object shall fail with a human readable error if a vals entry is supplied that is neither a valid postprocessor, scalar variable, function, or real number."
     issues = '#14169'
   [../]
+  
+  [./nl_vars]
+    type = RunException
+    input = 'nl_vars_error.i'
```

Comment:
you dont need this new input file since you are using the cli_args to pass the bad parameters anyway

Just use one of the others.